how-to-make-a-pitch-into-a-blogpost/readme.md

```diff
@@ -0,0 +1,51 @@
+>**Goal of this guide:** Explaining how to divide the process from idea to end result into small workable steps that are all individually publishable articles.
+
+# How to turn a pitch into a publishable article
+
+In this guide I will try to explain how to divide the process from idea to end result into small workable steps that are all individually publishable articles.
+
+## New Atoms
+
+Before we start the tricky stuff, you should first get a glimpse on how we work.
+At New Atoms everyone works asynchrone. This means that nothing you do can depend on something that came before or after it. This enables us to work in a process where everyone can do what they think they add the most value with to New Atoms at that moment, and everything we create is continuously integratable. This means that everything we do should add to the bigger whole and is able to stand on its own, no matter how small the implementation.
+
+To enable us to work this way we work with a [doables board](https://github.com/newatoms/guides/tree/ready/board-guide) which we fill with user stories, which are more in-depth version of todo post-its.
+
+## How to create and fulfill a user story for a blog post
+
+A userstory is meant to force you to clearly state what you try to achieve, how you want to do it and who it is for. You can read a more in depth guide about doables and user stories [here](https://github.com/newatoms/guides/tree/ready/board-guide).
+If you write a user story, keep the following mind:
+
+* All user stories are written from the perspective of the end user.
+* Collected information to fulfill a user story is a byproduct, never the goal on its own.
```

Comment:
Maybe 'byproduct' states too much that it is not that important. It can be very important.